Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38806&attempt_id=0c7345d5-f1da-4ae5-8757-ffc630089008)

Attempted: dragonfly-reverb

<details><summary>Partial log (click to expand)</summary><p>

```
make -C freeverb3
make[1]: Entering directory '/build/dragonfly-reverb/freeverb3'
gcc libsamplerate2/samplerate.c -Wall -Wextra -pipe -MD -MP -fPIC -DPIC -DNDEBUG -O2 -mtune=generic -msse -msse2 -fdata-sections -ffunction-sections -fvisibility=hidden -std=c99  -I. -DLIBSRATE2_FLOAT -c -o samplerate.c.o
gcc: error: unrecognized command line option '-msse'; did you mean '-fdse'?
gcc: error: unrecognized command line option '-msse2'
make[1]: *** [Makefile:27: samplerate.c.o] Error 1
make[1]: Leaving directory '/build/dragonfly-reverb/freeverb3'
make: *** [Makefile:14: libs] Error 2
builder for '/nix/store/l240nf1330286zhcr1k0v88nwdx8z68z-dragonfly-reverb-0.9.1.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/l240nf1330286zhcr1k0v88nwdx8z68z-dragonfly-reverb-0.9.1.drv' failed
```
</p></details>

--- a/pkgs/applications/audio/dragonfly-reverb/default.nix
+++ b/pkgs/applications/audio/dragonfly-reverb/default.nix
@@ -1,0 +1,33 @@
+{ stdenv, fetchgit, libjack2, libGL, pkgconfig, xorg }:
+
+stdenv.mkDerivation rec {
+  name = "dragonfly-reverb-${src.rev}";
+
+  src = fetchgit {
+    url = "https://github.com/michaelwillis/dragonfly-reverb";
+    rev = "0.9.1";
+    sha256 = "1dbykx044h768bbzabdagl4jh65gqgfsxsrarjrkp07sqnhlnhpd";
+  };
+
+  patchPhase = ''
+    patchShebangs dpf/utils/generate-ttl.sh
+  '';
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [
+    libjack2 xorg.libX11 libGL
+  ];
+
+  installPhase = ''
+    mkdir -p $out/lib/lv2/
+    cp -a bin/DragonflyReverb.lv2/ $out/lib/lv2/
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/michaelwillis/dragonfly-reverb;
+    description = "A hall-style reverb based on freeverb3 algorithms";
+    maintainers = [ maintainers.magnetophon ];
+    license = licenses.gpl2;
+    platforms = ["x86_64-linux"];
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -15269,6 +15269,8 @@ with pkgs;
 
   draftsight = callPackage ../applications/graphics/draftsight { };
 
+  dragonfly-reverb = callPackage ../applications/audio/dragonfly-reverb { };
+
   droopy = callPackage ../applications/networking/droopy {
     inherit (python3Packages) wrapPython;
   };

